README.md

```diff
@@ -74,7 +74,7 @@
 ## 🚀 Live Demo and video <a name="live-demo"></a>
 https://amazinggracee.github.io/Air-Quality/
```

Comment:
- [ ] Let's kindly make sure this project is properly deployed and accessible online all over the world. Right now this link is not working. You can use Render to deploy the project. All pages: home and detail page should work as expected :+1: